Sprint #19.adoc

```diff
@@ -23,6 +24,7 @@
 * TASK: handle a max size for upload
 * TASK: support inline attachment
 * TASK: generate mixed content (text + html)?
+* TASK: support attachments naming
```

Comment:
What about an attachment uploaded several times with different names ?
I really think the attachment name belongs the message, not the attachment